src/backend/web/routes/posts.js

```diff
@@ -1,6 +1,7 @@
 const express = require('express');
 const { getPosts } = require('../../utils/storage');
 const { logger } = require('../../utils/logger');
+const { count } = require('../../utils/storage.js');
```

Comment:
`const { getPostsCount } = require('../../utils/storage.js');`, there's nothing called `count` to get here.